contract/OffchainAggregatorBilling.sol

```diff
@@ -350,10 +412,10 @@ contract OffchainAggregatorBilling is Owned {
       address payee = s_payees[_transmitter];
       // Poses no re-entrancy issues, because LINK.transfer does not yield
       // control flow.
-      require(LINK.transfer(payee, linkWeiAmount), "insufficient funds");
+      require(s_linkToken.transfer(payee, linkWeiAmount), "insufficient funds");
```

Comment:
I don't know whether the solidity compiler is smart enough to do this for us: would it save any gas to read `s_linkToken` to a local variable, since we use it twice? (Here and on line 418?)